### PR TITLE
Config file support

### LINKED ISSUE #48: build.py looks for 'lyrics.git' but fails if the dir is named 'lyrics'
Fails with this error:

```
Traceback (most recent call last):
  File "../build.py", line 264, in <module>
    for letter in sorted(next(os.walk(sourceDatabaseDir))[1]):
StopIteration
make: *** [Makefile:20: build] Error 1
```

I think there should be a check in build.py to use either 'lyrics' or 'lyrics.git' (i.e. If 'lyrics.git' doesn't exist, try 'lyrics').

https://github.com/Lyrics/lyrics-website-generator/blob/4e0dad3afcb49fba0090cd7d3728df47b4b200d7/build.py#L14

For me, 'make' failed because my repo was cloned without the '.git' extension (and most all of my repos don't have a .git extension).

Another option, though may not be suitable for the size of this project, would be to add the lyrics db to this repo as a git submodule (which I could help with).

### PR DESCRIPTION
**Pull request by snshn**
Closes #49 
Closes #48

### CODE REVIEW

**Comment by snshn** [comment]
@andy5995 this should work if you modify config.ini after the first run (or just do `make config.ini` or `cp config.def.ini config.ini`)

File: Makefile [hunk 1]
```diff
@@ -43,3 +40,7 @@ serve: $(WWW_DIR)
         echo "Starting local server at http://0.0.0.0:8100" && \
         python3 -m http.server 8100
 .PHONY: serve
+
+install-dependencies:
+	pip3 install pystache simple-http-server
```
Comment (andy5995) [inline Makefile]:
```suggestion
	pip3 install --user pystache simple-http-server
```

**Comment by snshn** [comment]
@andy5995 please let me know if it's working well, I'll go ahead and merge it

**Review by andy5995 — APPROVED** [review]
Works fine!

File: README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
-This build tool compliments [lyrics.git](https://github.com/Lyrics/lyrics) to create static HTML files in `./www/`. The contents of that directory can then be uploaded to a server to become a fully functional lyrics website.
+Build static website out of [Open Lyrics Database](https://github.com/Lyrics/lyrics).
 
 
-## Requirements
+## Installing dependencies
 
     sudo apt-get install sassc
-    pip3 install pystache simple-http-server
```
Comment (andy5995) [inline README.md:9]: I actually liked it better the first way you had this, showing this part of the process clearly, and manually installing the packages with pip myself. imo, any developer (or even end-user) trying this shouldn't have any problem using pip separately from a makefile.

Comment (snshn) [reply README.md:9]: Yeah, it's a tricky one. I'm thinking that `sudo apt-get install sassc` should go into `make install-dependencies`, too... just with a smart way of determining what OS and what package manager is available, checking if it's already installed, etc...

Lemme roll it back, and then make another PR with that install-dependencies Makefile rule.

**Pull request merged**